content/en/docs/instrumentation/java/automatic/agent-config.md

```diff
@@ -65,7 +65,83 @@ You can enable [extensions][] by setting the corresponding property.
 
 ## Common instrumentation configuration
 
-See [common instrumentation configuration properties][common-config].
+Common settings that apply to multiple instrumentations at once.
+
+### Peer service name
+
+The [peer service name](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md#general-remote-service-attributes)
+is the name of a remote service to which a connection is made. It corresponds to `service.name` in
+the [Resource](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#service)
+for the local service.
+
+| System property / Environment variable | Description |
+| -------------------------------------- | ----------- |
+| `otel.instrumentation.common.peer-service-mapping` / `OTEL_INSTRUMENTATION_COMMON_PEER_SERVICE_MAPPING` | Used to specify a mapping from host names or IP addresses to peer services, as a comma-separated list of `<host_or_ip>=<user_assigned_name>` pairs. The peer service is added as an attribute to a span whose host or IP address match the mapping. For example, if set to `1.2.3.4=cats-service,dogs-abcdef123.serverlessapis.com=dogs-api`, requests to `1.2.3.4` will have a `peer.service` attribute of `cats-service` and requests to `dogs-abcdef123.serverlessapis.com` will have an attribute of `dogs-api`.
```

Comment:
@trask et all: The tables in the original page seemed unwieldy -- so little real estate was left to the **Description** column. So, in this version I combined the **System property** and **Environment variable** columns, separating the values with a `/`. Let me know what you think. (If you think that this works, I'll update the other tables in the page in the same way, but in a followup PR.)

Reply:
yeah, that's much better 👍

Reply:
Here's the tracking issue: #1247